index.html

```diff
@@ -1,7 +1,28 @@
 <!DOCTYPE html>
-<html>
-    <body>
-    <h1>My First Heading</h1>
-    <p>My first paragraph.</p>
-    </body>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles/styles.css">
+    <script src="app.js"></script>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>FLIC</title>
+</head>
+<body>
+
+    <!-- top banner -->
+    <!-- ADVERSTIMENET-->
```

Comment:
How are we going to make money with so few advertisements?

Please take this seriously.

Reply:
:+1: